google/cloud/sql/connector/InstanceConnectionManager.py

```diff
@@ -179,9 +179,8 @@ def __init__(
 
         self._user_agent_string = f"{APPLICATION_NAME}/{version}+{driver_name}"
         self._loop = loop
+        self._keys: Awaitable = keys
```

Comment:
This should have a specific return type: 
```suggestion
        self._keys: Awaitable[Type1, Type2] = keys
```

Reply:
Done